packages/lumx-react/src/components/thumbnail/Thumbnail.tsx

```diff
@@ -139,7 +139,23 @@ const Thumbnail: React.FC<ThumbnailProps> = ({
     focusPoint = DEFAULT_PROPS.focusPoint,
     ...props
 }: ThumbnailProps): ReactElement => {
-    const focusImageRef = useFocusedImage(focusPoint!);
+    const focusImageRef = useFocusedImage(focusPoint!, aspectRatio!, size!);
+
+    /**
+     * Check if browser is IE
+     * @return Browser is IE or not
+     */
+    const isInternetExplorer = () => {
+        const ua = window.navigator.userAgent;
+        const msie = ua.indexOf('MSIE ');
+        const isIEVersion = !!navigator.userAgent.match(/Trident.*rv\:11\./);
+
+        if (msie > 0 || isIEVersion) {
+            return true;
+        }
```

Comment:
could we just do `return msie > 0 || isIEVersion` ? just to make it a little bit more simpler